app/models/preupgrade_report_entry.rb

```diff
@@ -3,5 +3,11 @@
 class PreupgradeReportEntry < ApplicationRecord
   belongs_to :preupgrade_report
 
-  validates :preupgrade_report_id, :presence => true
+  validates :preupgrade_report_id, presence: true
```

Comment:
From [active record docs](https://guides.rubyonrails.org/active_record_validations.html#presence):

> If you want to be sure that an association is present, you'll need to test whether the associated object itself is present, and not the foreign key used to map the association. This way, it is not only checked that the foreign key is not empty but also that the referenced object exists.

Reply:
Also, multiple fields can be validated with single `validates`